dcicutils/ecr_utils.py

```diff
@@ -0,0 +1,68 @@
+import boto3
+import base64
+from .misc_utils import PRINT
+
+
+# Defines the standard Docker image tags for CGAP ECR
+# Latest and stable - something like this should be reasonable?
+CGAP_ECR_LAYOUT = {
+    'latest-wsgi': 'Latest version of WSGI Application',
+    'latest-indexer': 'Latest version of Indexer Application',
+    'latest-ingester': 'Latest version of the Ingester Application',
+    'stable-wsgi': 'Stable version of WSGI Application',
+    'stable-indexer': 'Stable version of Indexer Application',
+    'stable-ingester': 'Stable version of the Ingester Application'
+}
+
+
+class ECRUtils(object):
+    """ Utility class for interacting with ECR.
+        Initialized with an env name, from which a repo URL is resolved.
+
+        NOTE 1: unlike s3Utils, the resolved resources (repository) will NOT be
+        created if it does not already exist.
+        NOTE 2: the (already created) ECR repository must have the env_name in the URI
+    """
+
+    def __init__(self, *, env_name='cgap-mastertest', local_repository='cgap-wsgi'):
+        """ Creates an ECR client on startup """
+        self.env = env_name
+        self.local_repository = local_repository
+        self.is_cgap = 'cgap' in env_name
+        self.client = boto3.client('ecr', region='us-east-1')  # XXX: constant?
```

Comment:
I think the region should probably be an argument to the `__init__` function, no? You could make it default from some class variable.